Jenkinsfile

```
@@ -1 +1,9 @@
-buildPlugin(failFast: false, jdkVersions: [8, 11])
+buildPlugin(
+    failFast: false,
+    configurations: [
+        [ platform: 'linux', jdk: "8", jenkins: null ],
+        [ platform: 'windows', jdk: "8", jenkins: null ],
+        [ platform: 'linux', jdk: "11", jenkins: "2.163", javaLevel: "8" ],
```

Use 2.164 now? It is going to be the LTS baseline

```suggestion
        [ platform: 'linux', jdk: "11", jenkins: "2.164", javaLevel: "8" ],
```